airgun/entities/config_report.py

```diff
@@ -0,0 +1,67 @@
+from airgun.entities.base import BaseEntity
+from airgun.navigation import NavigateStep
+from airgun.navigation import navigator
+from airgun.utils import retry_navigation
+from airgun.views.config_report import ConfigReportDetailsView
+from airgun.views.config_report import ConfigReportsView
+
+
+class ConfigReportEntity(BaseEntity):
+    endpoint_path = '/config_reports'
+
+    def read(self, widget_names=None, host_name=None):
```

Comment:
I dont see a `hostname` used anywhere. Do you need to search for the hostname? then get that report isn't it ?

Reply:
Correct I am adding a search here like export for being specific for a host to read its values.
Nice catch!